src/screens/Books/index.js

```diff
@@ -28,14 +28,19 @@ class Books extends Component {
     }
   }
 
-  renderItem = ({ item }) => <ListItem elem={item} />;
+  renderItem = ({ item }) => {
+    const { navigation } = this.props;
```

Comment:
You can destructure all at once (if you're sure that the structure is already set), like this:

```js
/* This way, the only const you have is navigate */
const { navigation: { navigate } } = this.props;
```